Remotion/Core/Obsolete/ExtensibleEnums/ExtensibleEnum.cs

```diff
@@ -48,7 +48,7 @@ protected ExtensibleEnum (string id)
     }
 
     protected ExtensibleEnum (Type declaringType, string valueName)
-        : this (declaringType.FullName, valueName)
+        : this (declaringType.GetFullNameSafe(), valueName)
```

Comment:
I just discovered that this project is not built and currrently does not compile because there is no reference to Core. Should I revert all the changes in the file?

Reply:
oh, yes. the obsolete-projects can be reverted. they don't get api changes or NRT. they only exist to support dependdb lookups.